apps/web/src/locales/en/messages.json

```
@@ -4688,6 +4688,71 @@
     "message": "or",
     "description": "This will be used as part of a larger sentence, broken up to include links. The full sentence will read 'Learn more, see how it works, **or** try it now.'"
   },
+  "developmentDevOpsAndITTeamsChooseBWSecret": {
+    "message": "Development, DevOps, and IT teams choose Bitwarden Secrets Manager to securely manage and deploy their infrastructure and machine secrets."
+  }
+  ,
+  "centralizeSecretsManagement": {
+    "message": "Centralize secrets management."
+  }
+  ,
+  "centralizeSecretsManagementDescription": {
+    "message": "securely store and manage secrets in one location to prevent secret sprawl across your organization."
+  }
+  ,
+  "preventSecretLeaks": {
+    "message": "Prevent secret leaks."
+  }
+  ,
+  "preventSecretLeaksDescription": {
+    "message": "Protect secrets with end-to-end encryption. No more hard coding secrets or sharing through env. files."
+  }
+  ,
+  "enhanceDeveloperProductivity": {
+    "message": "Enhance developer productivity."
+  }  
+  ,
+  "enhanceDeveloperProductivityDescription": {
+    "message": "Programmatically retrieve and deploy secrets at runtime so developers can focus on what matters most, like improving code quality."
+  }
+  ,
+  "strengthenBusinessSecurity": {
+    "message": "Strengthen business security."
+  }
+  ,
```

formatting of the closing bracket and comma (from 4691 line until here)
```suggestion
  },
```

---

thanks!